src/main/java/janggi/point/Point.java

```diff
@@ -0,0 +1,52 @@
+package janggi.point;
+
+public record Point(int row, int column) {
+
+    private static final int LOWER_BOUND = 0;
+    private static final int ROW_UPPER_BOUND = 9;
+    private static final int COLUMN_UPPER_BOUND = 8;
+
+    public Point(int row, int column) {
+        validateRange(row, column);
+        this.row = row;
+        this.column = column;
+    }
```

Comment:
```suggestion
    public Point {
        validateRange(row, column);
    }
```
`record class`의 축약 생성자를 활용할 수 있습니다.